docs/source/cheat_sheet.rst

```diff
@@ -120,6 +120,13 @@ When you're puzzled or when things are complicated
    # dynamic to write a type for.
    x = mystery_function() # type: Any
 
+   # This is how to deal with kwargs, generally just use Any
```

Comment:
I'd call these "varargs" -- "kwargs" is ambiguous as it could also refer to individually named keyword args (e.g. line 75 above). Also I disagree with the recommendation to use `Any`, it's more illustrative to show e.g. `str` and then explain that each of the args must be a string. E.g. with
```python
def foo(*args):
    # type: (str) -> str
    # The type of args in the body is Tuple[str, ...]
    return ','.join(args)
```